packages/react/src/lib/accessibility/FocusZone/CHANGELOG.md

```diff
@@ -26,6 +26,12 @@ This is a list of changes made to this Stardust copy of FocusZone in comparison
 - Handle keyDownCapture based on `shouldHandleKeyDownCapture` prop @sophieH29 ([#563](https://github.com/stardust-ui/react/pull/563))
 - Add `bidirectionalDomOrder` direction allowing arrow keys navigation following DOM order @sophieH29 ([#1637](https://github.com/stardust-ui/react/pull/1647))
 
+### Upgrade `FocusZone` to the latest version from `fabric-ui` @sophieH29 ([#1772](https://github.com/stardust-ui/react/pull/1772))
+- Restore focus on removing item ([OfficeDev/office-ui-fabric-react#7818](https://github.com/OfficeDev/office-ui-fabric-react/pull/7818))
```

Comment:
I don't see any UTs added for 7818.

Reply:
There are tests https://github.com/stardust-ui/react/pull/1772/files#diff-108722f1de4d58a5ec0ea27e886d63d1R1568